lib/datasets/wine.rb

```diff
@@ -0,0 +1,64 @@
+require 'csv'
+
+require_relative 'dataset'
+
+module Datasets
+  class Wine < Dataset
+    Record = Struct.new(:wine_class,
+                        :alcohol,
+                        :malic_acid,
+                        :ash,
+                        :alcalinity_of_ash,
+                        :magnesium,
+                        :total_phenols,
+                        :flavanoids,
```

Comment:
Does the field store "the number of flavonoids"?
If so, `n_flavonoids` is better.

By the way, `flavanoides` (`va` -> `vo`) is typo?

Reply:
I see.

I copied attribute names from http://archive.ics.uci.edu/ml/machine-learning-databases/wine/wine.names, but here is `flavanoides ` ...

Reply:
I will fix it, but spell "flavanoids" seems not to be typo. https://ejje.weblio.jp/content/flavanoids

Reply:
fixed

Reply:
I see.